tox.ini

```diff
@@ -1,10 +1,20 @@
 [tox]
-envlist = py27, py34, py35, py36, py37, py38, pypy, pypy3.5
+envlist = 
+    py{27, 34, 35, 36, 37, 38, py, py3.5}
+    flake8
 skipsdist = True
+skip_missing_interpreters = true
 
 [testenv]
 usedevelop = True
-commands = nosetests -v --nocapture {posargs}
+commands = nosetests --with-coverage -v --nocapture {posargs}
```

Comment:
The recommended way to use coverage.py and nose is documented at https://coverage.readthedocs.io/en/latest/faq.html

We also need to create a `.coveragerc` file to 1) exclude some files (for example `astor/codegen.py`) from the report 2) add missing branch information to the report.